contributors/devel/kubemark-guide.md

```diff
@@ -43,62 +51,83 @@ Common workflow for Kubemark is:
 - monitoring test execution and debugging problems
 - turning down Kubemark cluster
 
-Included in descriptions there will be comments helpful for anyone who’ll want to
-port Kubemark to different providers.
+(For now) Included in descriptions there will be comments helpful for anyone who’ll
+want to port Kubemark to different providers.
+(Later) When the refactoring mentioned in the above section finishes, we would replace
+these comments with a clean API that would allow kubemark to run on top of any provider.
 
 ### Starting a Kubemark cluster
 
 To start a Kubemark cluster on GCE you need to create an external kubernetes
 cluster (it can be GCE, GKE or anything else) by yourself, make sure that kubeconfig
 points to it by default, build a kubernetes release (e.g. by running
 `make quick-release`) and run `test/kubemark/start-kubemark.sh` script.
-This script will create a VM for master components, Pods for HollowNodes
-and do all the setup necessary to let them talk to each other. It will use the
-configuration stored in `cluster/kubemark/config-default.sh` - you can tweak it
-however you want, but note that some features may not be implemented yet, as
-implementation of Hollow components/mocks will probably be lagging behind ‘real’
-one. For performance tests interesting variables are `NUM_NODES` and
-`MASTER_SIZE`. After start-kubemark script is finished you’ll have a ready
-Kubemark cluster, a kubeconfig file for talking to the Kubemark cluster is
-stored in `test/kubemark/kubeconfig.kubemark`.
-
-Currently we're running HollowNode with limit of 0.05 a CPU core and ~60MB or
-memory, which taking into account default cluster addons and fluentD running on
-an 'external' cluster, allows running ~17.5 HollowNodes per core.
+This script will create a VM for master (along with mounted PD and firewall rules set),
+then start kubelet and run the pods for the master components. Following this, it
+sets up the HollowNodes as Pods on the external cluster and do all the setup necessary
+to let them talk to each other. It will use the configuration stored in
+`cluster/kubemark/config-default.sh` - you can tweak it however you want, but note that
+some features may not be implemented yet, as implementation of Hollow components/mocks
+will probably be lagging behind ‘real’ one. For performance tests interesting variables
+are `NUM_NODES` and `KUBEMARK_MASTER_SIZE`. After start-kubemark script is finished,
+you’ll have a ready Kubemark cluster, and a kubeconfig file for talking to the Kubemark
+cluster is stored in `test/kubemark/kubeconfig.kubemark`.
+
+Currently we're running HollowNode with a limit of ~0.1 CPU core / pod and ~[FILL]MB of
+memory, taking into account the default cluster addons and fluentD running on the
```

Comment:
This sentence is misleading. We have 0.09 cpu per HollowNode NOT including cluster addons. Since the later depends on machine sizes, etc., we shouldn't mix this together but say it explicitly.